pom.xml

```diff
@@ -148,20 +148,21 @@
     <closure.version>v20130603</closure.version>
     <wro4j.version>1.7.0</wro4j.version>
     <glassfish-javax-json.version>1.0.1</glassfish-javax-json.version>
-    <groovy.version>1.8.5</groovy.version>
+    <groovy.version>2.4.15</groovy.version>
     <guava.version>11.0</guava.version>
-    <eclipselink.version>2.5.1</eclipselink.version>
+    <eclipselink.version>2.7.4-SNAPSHOT</eclipselink.version>
```

Comment:
This pre-release version is necessary for getting the static weaving to work with Java 11.